source/conf.py

```diff
@@ -78,6 +78,10 @@ def setup(_: Sphinx):
         "https://docs.mattermost.com/about/subscription.html#do-you-have-a-program-for-official-non-profits-open-source-projects-and-charities",
 "about/deployments-and-editions.html":
         "https://docs.mattermost.com/about/editions-and-offerings.html",
+"https://docs.mattermost.com/about/faq-business.html#copyright-and-licensing-of-mattermost-server-open-source-code":
```

Comment:
Sorry, one more 🙏 Would we also add the following redirect @justinegeffen ?

```
"https://docs.mattermost.com/about/faq-mattermost-source-available-license.html":
	"http://docs.mattermost.com/about/faq-license.html#source-available-licensing",
```

Reply:
Thanks, Jason! Appreciate the catch (and formatting). :)